plugins/pulp_docker/plugins/registry.py

```diff
@@ -448,7 +450,7 @@ def get_tags(self):
                                                      repo=self.name,
                                                      registry=self.registry_url,
                                                      reason=str(e))
-        return json.loads(tags)['tags']
+        return json.loads(tags)['tags'] or []
```

Comment:
The docker repository I was testing with had tags: null, this breaks pulp_docker at https://github.com/pulp/pulp_docker/blob/master/plugins/pulp_docker/plugins/importers/sync.py#L228 

Having a null tag is a bug: https://github.com/docker/distribution/issues/1186, but since the fix is very simple I added it as a defensive measure

Reply:
i was able to find repo with tags and tested the sync with osbs-test/additional-tags

Reply:
How do you feel about leaving this in so pulp won't barf at this in the future; especially since the fix is very simple on our end?

Reply:
oh no, i did not intended to remove this, just confirmed that your fix is working with repo with and without tags